bundles/org.openhab.binding.modbus.lambda/src/main/resources/OH-INF/thing/lambda-heatpump-channel-types.xml

```diff
@@ -0,0 +1,430 @@
+		<label>Calculated Ambient Temperature</label>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="emanager-error-number-type">
+		<item-type>Number</item-type>
+		<label>E-Manager Error Number</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">No Error</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="emanager-operating-state-type">
+		<item-type>Number</item-type>
+		<label>E-Manager Operating State</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">OFF</option>
+				<option value="1">AUTOMATIC</option>
+				<option value="2">MANUAL</option>
+				<option value="3">ERROR</option>
+				<option value="4">OFFLINE</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="actual-power-type">
+		<item-type unitHint="%">Number:Power</item-type>
+		<label>Actual Power (input or excess)</label>
+		<state readOnly="false" pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="actual-power-consumption-type">
+		<item-type unitHint="%">Number:Power</item-type>
+		<label>Actual Power Consumption</label>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="power-consumption-setpoint-type">
+		<item-type unitHint="%">Number:Power</item-type>
+		<label>Power consumption setpoint</label>
+		<state readOnly="true" pattern="%.0f %unit%"/>
+	</channel-type>
+
+	<channel-type id="heatpump1-error-state-type">
+		<item-type>Number</item-type>
+		<label>Heatpump1 Error State</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">NONE</option>
+				<option value="1">MESSAGE</option>
+				<option value="2">WARNING</option>
+				<option value="3">ALARM</option>
+				<option value="4">FAULT</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="heatpump1-error-number-type">
+		<item-type>Number</item-type>
+		<label>Heatpump1 Error Number</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">No Error</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="heatpump1-state-type">
+		<item-type>Number</item-type>
+		<label>Heatpump1 State</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">INIT</option>
+				<option value="1">REFERENCE</option>
+				<option value="2">RESTART-BLOCK</option>
+				<option value="3">READY</option>
+				<option value="4">START PUMPS</option>
+				<option value="5">START COMPRESSOR</option>
+				<option value="6">PRE-REGULATION</option>
+				<option value="7">REGULATION</option>
+				<option value="8">Not used</option>
+				<option value="9">COOLING</option>
+				<option value="10">DEFROSTING</option>
+				<option value="20">STOPPING</option>
+				<option value="30">FAULT-LOCK</option>
+				<option value="31">ALARM-BLOCK</option>
+				<option value="40">ERROR-RESET</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="heatpump1-operating-state-type">
```

Comment:
Applies to all channel types with the id prefix: `heatpump1` the 1 can be removed. As it is the type and the type is used for _all_ heatpumps.